### DISCUSSION TITLE
Migration from Xamarin.Forms Prism to Maui Prism: NavigationBar is gone, expect for dialogs ...

### DISCUSSION THREAD

**Discussion by tommy-uytterhaegen** [PrismLibrary/Prism#3279] [.NET MAUI]
I'm working on bringing an existing project from XamarinForms to Maui. When working on the prism conversion everything works fine except for the fact that I don't have a NavigationBar ... 

I can use the back functionality on Android, and that works as expected, but my screen looks like a 'NavigationPage.HasNavigationBar = false' (and no, I did not set that variable and yes, I have tried setting it to 'true' :))

When opening dialogs (with Mopup) it does show the NavigationBar and I have to hide them.

I start from a TabbedPage (this has a NavigationBar where I see the title & have icons). This TabbedPage has a couple of children (tabs) and each child tab page is wrapped in a NavigationPage. As soon as I navigate to a new page, this page goes fullscreen and I lose the NavigationBar.

I hope someone can point me in the right direction because it is driving me crazy :)

Some code:

```
<TabbedPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
                       xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                       xmlns:views="clr-namespace:Tuyware.MyBoardGameCollection.Views;assembly=Tuyware.MyBoardGameCollection.Maui"
                       xmlns:viewmodels="clr-namespace:ViewModels"
                       x:Class="Views.MainPage"  
            
                       xmlns:android="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;assembly=Microsoft.Maui.Controls"
                       android:TabbedPage.ToolbarPlacement="Bottom"
            
                       xmlns:prism="http://prismlibrary.com"
                       prism:ViewModelLocator.AutowireViewModel="Forced"
            
                       x:DataType="viewmodels:MainViewModel">
    
    <TabbedPage.Children>

        <NavigationPage>
            <x:Arguments>
                <views:ChildAPage />
            </x:Arguments>
        </NavigationPage>

        <NavigationPage>
            <x:Arguments>
                <views:ChildBPage />
            </x:Arguments>
        </NavigationPage>

    </TabbedPage.Children>
    
</TabbedPage>
```

I call new pages with:

`await NavigationService.NavigateAsync(pageToGoToName, navigationParameters);`

All pages are 'RegisterForNavigation' with their ViewModel

Any ideas?

**Comment by AngeloLazzari** [comment]
Did you solve this?

**Reply by tommy-uytterhaegen** [reply]
No, and since there was no support, I decided to completely strip the library out of my code, which went better than expected :)
